charts/ecosystem/values.yaml

```diff
@@ -48,6 +48,34 @@ maxEngines: "10"
 galasaWebUiImage: "galasa-ui"
 #
 #
+#
+# Optional. The label of a Kubernetes node that the engine controller should
+# have an affinity for. If possible, the engine controller will schedule
+# Galasa runs on this node.
+#
+# Example value: "galasa-engines"
+#
+k8s_node_preferred_affinity: ""
+#
+#
+#
+# Optional. The definition of a Kubernetes node taint toleration behaviour.
+# You may wish to prevent other workloads from being scheduled on your dedicated
+# Galasa k8s nodes. This can be achieved by tainting the Galasa k8s nodes. If this
+# has been done, you will need to define a toleration for your Galasa pods so that
+# they can schedule on the tainted nodes.
+#
+# The value should define a node label, an operator and a condition.
+#
+# Example value: "galasa-engines=Exists:NoSchedule"
+#
+# In the above example, any node which has the label "galasa-engines" with a
+# "NoSchedule" taint should be tolerated, resulting in the Galasa test pods
+# having exclusive access to run on this node.
+#
+k8s_node_tolerations: ""
```

Comment:
The values are camelCase but then the config.yaml file for the ecosystem seems to be snake_case, is it worth enforcing some overall consitancy across the helm repo?

Reply:
I've gone for camelcase for the values in values.yaml in the newest push

Reply:
> The values are camelCase but then the config.yaml file for the ecosystem seems to be snake_case, is it worth enforcing some overall consitancy across the helm repo?

Ideally yes, it would be good for the values in config.yaml to also be consistent with the rest of the code, but that would require more code changes in the k8s controller.

Thanks for making the change - the references to the camelCase values in the config.yaml seem like they still need updating, for example:

```yaml
galasa_node_preferred_affinity: "{{ .Values.k8sNodePreferredAffinity }}"
galasa_node_tolerations: "{{ .Values.k8sNodeTolerations }}"
```